index.js

```diff
@@ -96,6 +96,8 @@ function toRunCommand (inspectObj, name) {
     })
   }
 
+  if (hostcfg.ExtraHosts) rc = appendJoinedArray(rc, '--add-host', hostcfg.ExtraHosts, ' ')
```

Comment:
I was finally able to test this, and it doesn't work correctly for multiple entries in `hostcfg.ExtraHosts` (it produces `--add-host "xyz:1.2.3.4 abc:5.6.7.8"` instead of the desired `--add-host xyz:1.2.3.4 --add-host abc:5.6.7.8`).

Fortunately it's an easy fix. Just change the code to this:

```js
rc = appendArray(rc, '--add-host', hostcfg.ExtraHosts)
```